gunpowder/nodes/simple_augment.py

```diff
@@ -53,7 +53,7 @@ def setup(self):
 
     def prepare(self, request):
 
-        self.total_roi = request.get_total_roi()
```

Comment:
I think we do want to modify in place here. The way prepare works is it merges the request returned from prepare with the original request that was given. This is very convenient in most cases since we usually don't want to mess with the requests made from nodes lower in the tree, but here we don't want to request both the mirrored/transposed array and the original, this could lead to excessively large requests.